app/Http/Controllers/Backend/Export/ExportController.php

```diff
@@ -24,7 +24,16 @@ abstract class ExportController extends Controller
      * @throws DataOverflowException If too many results are given.
      */
     public static function getExportableStatuses(User $user, Carbon $timestampFrom, Carbon $timestampTo): Collection {
-        $statuses = Status::join('train_checkins', 'statuses.id', '=', 'train_checkins.status_id')
+        $statuses = Status::with([
+                                     'trainCheckin',
+                                     'trainCheckin.HafasTrip',
```

Comment:
This line can be removed because the third line also loads this relations. This saves 2 queries. :)
```suggestion
```